### ISSUE TITLE
Change outDir or declarationDir when issue is fixed on ATL

### ISSUE THREAD

**Issue by alexjoverm** [alexjoverm/typescript-library-starter#19]
Relates to #18 

That was fixed temporally, but it would be nicer to generate types under a folder called `types` instead of `src.

This is dependant on a fix on ATL end.

**Comment by alexjoverm** [comment]
Opened https://github.com/s-panferov/awesome-typescript-loader/issues/388

**Comment by superamadeus** [comment]
I don't know if I'm not understanding the issue correctly, but the following changes to tsconfig.json generates the the types in 'dist/types'. Please note that I am on Mac OSX, and I have not tested this in any other environment.

### Changes
 - change _outDir_ to "dist/types".
 - add "src" to the _include_ array

### Complete `tsconfig.json`:

```json
{
    "compilerOptions": {
        "moduleResolution": "node",
        "target": "es2015",
        "lib": ["es2016", "dom"],
        "noImplicitAny": true,
        "strictNullChecks": true,
        "sourceMap": true,
        "declaration": true,
        "outDir": "dist/types",
        "typeRoots": [
          "node_modules/@types"
        ]
    },
    "include": [
        "src"
    ],
    "awesomeTypescriptLoaderOptions": {
        "useBabel": true
    }
}
```

**Comment by alexjoverm** [comment]
Fixed by #23
